recipe/meta.yaml

```diff
@@ -10,17 +10,15 @@ package:
 
 source:
   git_url: https://github.com/openmm/{{ name }}.git
-  git_rev: "994f92fde4ce710a4db5557b6bc22756d5f9d2b1"
+  git_rev: "e196c8d52791e5b6e2e90b96e11a37893883e2f4"
```

Comment:
We should switch to using tarballs if we can

Reply:
That should help with stuff like this https://github.com/openmm/openmm-torch/issues/113#issuecomment-1648103226 I think when I first made this feedstock, we didn't have a release yet and wanted to just test it which is why I used `git_url` but `url` is recommended

Reply:
We should always build conda packages *before* creating the github release.  Because very often, when we try to build the packages, we discover a problem that requires further code changes.  Until we've successfully built packages, we don't know for sure whether the code is final.  That's why we have to do it by git revision.

Reply:
We could do either do an `rc` or do a PR to test it using the git revision, and if that passes, then change it to use a tarball for the actual release if we need to test it. I know that exceptions can be made (especially if there are git submodules (but then really this method should be used https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#source-from-multiple-sources)

The docs are pretty explicit about the preference for tarballs https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos and it is less likely we get out of sync with the repository. 

Is there an issue with this workflow?

* Decide we want to gut a release
* open PR and use `git_url` to see if package builds
* once it builds/changes upstream are made to enable building, cut a release on github
* change pr to use `url` for source 
* merge

I believe that covers the issue of needing to change upstream for a conda-forge release, while also address the preference to build packages from tarballs.

Reply:
The first reason they give for preferring tarballs is

> Repositories are usually larger than tarballs, draining shared CI time and bandwidth

That workflow would involve doing an extra build, after switching to the tarball.  So I think it would consume more CI time, not less?

Reply:
OK! We follow the conda-forge rules (https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)